### PR TITLE
Update driver to 2.0.7

### LINKED ISSUE #4: Update AthenaJDBC version to 2.0.7
Add Driver Configuration Options added in AthenaJDBC 2.0.6 and 2.0.7

new feature

- SAML authentication
- Workgroup
- CATS statements

etc

[Simba Athena JDBC Driver with SQL Connector](https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC_2.0.7/docs/Simba+Athena+JDBC+Driver+Install+and+Configuration+Guide.pdf)

### PR DESCRIPTION
**Pull request by matsuiyuki**
close #4 

- fix and bump sbt version
- add driver config option

### CODE REVIEW

**Comment by matsuiyuki** [comment]
@zaneli Now that the source code has been completed, I would like you to review my code.

The order of additional options has been adapted to the document.

File: src/main/scala/scalikejdbc/athena/Config.scala [hunk 1]
```diff
@@ -81,17 +82,24 @@ object Config {
...
   val MaxColumnNameLength = "MaxColumnNameLength"
   val MaxErrorRetry = "MaxErrorRetry"
+  val MaxQueryExecutionPollingInterval = "MaxQueryExecutionPollingInterval"
   val MaxSchemaNameLength = "MaxSchemaNameLength"
+  val MaxStreamErrorRetry = "MaxStreamErrorRetry"
   val MaxTableNameLength = "MaxTableNameLength"
   val MetadataRetrievalMethod = "MetadataRetrievalMethod"
   val NonProxyHosts = "NonProxyHosts"
+  val Password = "Password"
   val PWD = "PWD"
```
Comment (zaneli) [inline src/main/scala/scalikejdbc/athena/Config.scala:99]: `PWD` and `UID` seems to have been removed...? 🤔

Oh, I found [the following description](https://s3.amazonaws.com/athena-downloads/drivers/JDBC/SimbaAthenaJDBC_2.0.7/docs/Simba+Athena+JDBC+Driver+Install+and+Configuration+Guide.pdf).

* Password

> As an alternative, you can configure this property using the alias `PWD`.

* User

> As an alternative, you can configure this property using the alias `UID`.

It is desirable to leave these. 👍

**Review by zaneli — APPROVED** [review]
Great work! 🥇

**Comment by zaneli** [comment]
@matsuiyuki 
Thank you!
I'll release scalikejdbc-athena 0.2.1.

**Pull request merged**